core/src/main/scala/org/apache/spark/scheduler/EventLoggingListener.scala

```diff
@@ -86,8 +86,14 @@ private[spark] class EventLoggingListener(
    * Creates the log file in the configured log directory.
    */
   def start() {
-    if (!fileSystem.isDirectory(new Path(logBaseDir))) {
-      throw new IllegalArgumentException(s"Log directory $logBaseDir does not exist.")
+    try{
```

Comment:
Ok, got it, sorry and thanks for correcting it. I read the style guid last time I contributed, but obviously forgot and didn't look at it again this time. However, I did use the scalastyle script in the dev folder this time and it said everything was good, so I probably need to figure out what I did wrong with that.